drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/InMemoryMigrationStrategy.java

```diff
@@ -22,6 +22,11 @@
 
     @Override
     public Function<String, String> from1_0to1_1() {
-        return rawXml -> rawXml.replaceAll("EXPECTED", "EXPECT");
+        return rawXml -> rawXml.replaceAll("EXPECTED", "EXPECT").replaceAll("<ScenarioSimulationModel version=\"1.0\">", "<ScenarioSimulationModel version=\"1.1\">");
+    }
+
+    @Override
+    public Function<String, String> from1_1to1_2() {
+        return rawXml -> rawXml.replaceAll("</simulationDescriptor>", "<dmoSession></dmoSession>\n<type>RULE</type>\n</simulationDescriptor>").replaceAll("<ScenarioSimulationModel version=\"1.1\">", "<ScenarioSimulationModel version=\"1.2\">");
```

Comment:
I think we should implement the migration strategy to be idempotent, at the moment if (for any reason) it will be called multiple times it will add the missing nodes every time, maybe we can implement an `if` to skip if already available. WDYT?

Reply:
@danielezonca 
for the moment being, not having a proper xml dom/bean to manage, the only thing I can do is to check for already existing tag - better then nothing, but not really nice.